Feature/new home page

erinwilhjelm opened the pull request.

erinwilhjelm committed:
updated home page

erinwilhjelm committed:
new landing page

Commit:
Merge remote-tracking branch 'origin/develop' into feature/NewHomePage

ashuping reviewed (changes requested):
We should avoid absolute positioning when possible - it doesn't zoom well. Also, it looks like you went back to increasing the border size rather than changing the color? I decided on changing the color because growing the borders pushes the icon/text down, and can make the text wrapping change if the service name is a certain length.

Commit:
Modify css to remove absolute positioning

~ Modify CategoryPage.css to remove absolute positioning and handle element positioning relatively instead.
~ Modify CategoryPage.js to better support relative positioning.
~ Modify CategoryPage.css to use color-change instead of margin-growth to indicate hovered-over buttons.

Commit:
Add maximum width and center cat-page header

+ Add maximum-width element. When the page grows beyond this maximum width, it is constrained in the center.
~ Modify CategoryPage.css so that the cat-page header image and text look better on narrow screens.

hjarrell commented on client/src/App.js:
```diff
@@ -14,11 +13,10 @@ const App = () => {
+        <Route exact path="/cat" component={CategoryPage} />
         <Route exact path="/">
-          <Redirect to="/home" />
+          <Redirect to="/cat" />
```
Can we change this from /cat to just / ? I think that it would be more professional and look better.

ashuping replied:
Done.

Commit:
Place CategoryPage at the server-root.

Commit:
Merge remote-tracking branch 'origin/develop' into feature/NewHomePage
